src/__tests__/use-stepper.test.tsx

```diff
@@ -66,59 +61,27 @@ describe('useStepper', () => {
 
   it('returns the correct properties', () => {
     const { result } = renderHook(() => useStepper());
-    expect(result.current).toMatchInlineSnapshot(`
-      {
-        "decrement": [Function],
-        "getDecrementProps": [Function],
-        "getFormProps": [Function],
-        "getIncrementProps": [Function],
-        "getInputProps": [Function],
-        "increment": [Function],
-        "setValue": [Function],
-        "value": "0",
-      }
-    `);
+    expect(result.current).toMatchSnapshot();
```

Comment:
I prefer inline snapshots, please revert and update the inline snapshot. (Note: you can just delete the inline snapshot string and re-run the tests to generate an updated version.)